### PR TITLE
Added Facebook Open Graph metatags

### PR DESCRIPTION
**Pull request by mikylucky**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,16 @@
   <link rel="icon" href="/logo.png">
   <link rel="manifest" href="/manifest.webmanifest">
   <title>Kanttiinit</title>
+
+  <!-- Facebook Meta tags -->
+  <meta property="og:url"                content="http://www.kanttiinit.fi" />
+  <meta property="og:type"               content="website" />
+  <meta property="og:title"              content="Kanttiinit" />
+  <meta property="og:description"        content="Student lunch effortlessly" />
```
Comment (bodyflex) [inline src/index.html]: I think our official slogan should end with a period "Student lunch effortlessly."

**Comment by mikylucky** [comment]
I just noticed we have a language problem. Being the url the same for the english version and the finnish version we have to choose one of the two for the Facebook preview. Which one do you prefer?

**Comment by bodyflex** [comment]
let's go with english for now.

but that's actually a good point, we might want to consider different URLs for different languages in the future!

**Comment by mikylucky** [comment]
If you plan to add more urls to the client, than you should move the meta tags in the React code in order to make them dynamic (I cannot code in React). I'm adding the final dot to the sentence, further developments up to someone React speaking :D

**Pull request merged**